docs/contribution_guide/releases.rst

```diff
@@ -0,0 +1,55 @@
+Creating new releases
+---------------------
+
+First add a description of the changes introduced in the package version you
+want to release to `CHANGES.md <https://github.com/fairlearn/fairlearn/CHANGES.md>`_.
+
+It is also best to verify that the Dashboard loads correctly. This is slightly
+involved:
+
+#. Create a wheel by running :code:`python setup.py sdist bdist_wheel` from
+   the repository root. This will create a :code:`dist` directory which
+   contains a :code:`.whl` file.
+#. Create a new conda environment for the test
```

Comment:
Not everybody is using `conda`. Please link to how to do this in conda, but also using [venv](https://docs.python.org/3/library/venv.html).

Reply:
@MiroDudik this is automated using conda. I don't think we need to provide alternatives for something that ideally nobody should be doing manually anyway. Makes sense?

Reply:
What I was suggesting is something like
```
#. Create a new python environment for the test, e.g.,
```
and give the one liners how to do it in conda and venv (or links to how to do it in conda or venv).

Reply:
I understand. These are instructions for doing a release, though. Other than maintainers nobody will ever have to do this. Even for maintainers this is already automated. I'd rather focus on sections that are actually relevant to users/contributors.

Reply:
Makes sense. I'll fix this in a future PR (the reason is that I don't use conda and I always need to look up in my cheat sheet the two liner how to set up a new environment with venv)